src/controlFlowCodingPractice_15March2023/FactorialOfNumber.java

```diff
@@ -0,0 +1,22 @@
+package controlFlowCodingPractice_15March2023;
+
+import java.util.Scanner;
+
+public class FactorialOfNumber {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Scanner console = new Scanner(System.in);
```

Comment:
remove the scanner

Reply:
Removed scanner class and pushed the changes